src/stories/OSCALMetadata.stories.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import OSCALMetadata from "../components/OSCALMetadata";
+
+export default {
+  title: "Components/Metadata",
+  component: OSCALMetadata,
+};
+
+const Template = (args) => <OSCALMetadata {...args} />;
+
+export const Default = Template.bind({});
+
+export const MetadataParties = Template.bind({});
+
+export const MetadataRoles = Template.bind({});
```

Comment:
I think we can combine `MetadataParties` and `MetadataRoles` into a single `WithPartiesAndRoles`.